theme/acquia_lift.wizard.theme.inc

```diff
@@ -45,6 +46,9 @@ function theme_acquia_lift_card($variables) {
   }
   $html .= '</div>';
   $html .= '<div class="el-card__content">';
+  if (!empty($card_flag)) {
```

Comment:
In the case of dragons there really are two cards active at a time, the one you are coming from and your possible target.... Specifically to the flag, more than one card could have it at a time.